modules/dxp/apps/segments/segments-experiment-web/test/js/components/SegmentsExperimentsSidebar.es.js

```diff
@@ -620,7 +620,7 @@ describe('Winner declared', () => {
 		expect(publishExperience).toHaveBeenCalledTimes(0);
 	});
 
-	it('Discard button action for winner declared status', async () => {
+	it.skip('Discard button action for winner declared status', async () => {
```

Comment:
1/2 I'm skipping this test because it's related to the discard test button. Considering we're planning to remove it in the future and keep only the trash icon in the sidepanel (as we discussed this week), it does not make sense keep it this test.

Reply:
Hey @fmarcio you can remove this unit test.

Reply:
done!